pkg/handler/collector/deps_dev/deps_dev_test.go

```diff
@@ -38,6 +35,10 @@ import (
 	"github.com/guacsec/guac/pkg/events"
 	"github.com/guacsec/guac/pkg/handler/collector"
 	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/guacsec/guac/pkg/logging"
+
+	pb "deps.dev/api/v3"
+	"github.com/google/go-cmp/cmp"
```

Comment:
can we add a test to make sure that a ratelimit fails gracefully?

Reply:
I have included tests for osv, deps dev, and clearly defined.